diff --git a/usb/usb-skeleton.c b/usb/usb-skeleton.c
index e24ce31..0ae1423 100644
--- a/usb/usb-skeleton.c
+++ b/usb/usb-skeleton.c
@@ -27,9 +27,13 @@
 #define USB_SKEL_VENDOR_ID	0xfff0
 #define USB_SKEL_PRODUCT_ID	0xfff0
 
+#define USB_SKEL_G_ZERO_VID 0x0525
+#define USB_SKEL_G_ZERO_PID 0xa4a0
+
 /* table of devices that work with this driver */
 static const struct usb_device_id skel_table[] = {
 	{ USB_DEVICE(USB_SKEL_VENDOR_ID, USB_SKEL_PRODUCT_ID) },
+	{ USB_DEVICE(USB_SKEL_G_ZERO_VID, USB_SKEL_G_ZERO_PID) },
 	{ }					/* Terminating entry */
 };
 MODULE_DEVICE_TABLE(usb, skel_table);
@@ -595,7 +599,7 @@ static int skel_probe(struct usb_interface *interface,
 
 	/* let the user know what node this device is now attached to */
 	dev_info(&interface->dev,
-		 "USB Skeleton device now attached to USBSkel-%d",
+		 "USB Skeleton device now attached to USBSkel-%d\n",
 		 interface->minor);
 	return 0;
 
